tests/vulkansc/negative/pipeline.cpp

```diff
@@ -0,0 +1,400 @@
+        created_pipelines = create_pipelines(m_device->handle(), pipeline_layout.handle(), render_pass.handle(),
+            {size1, size2, size1, size1, size1, size2, size1, size1},
+            {capacity_vuid});
+        pipelines.insert(pipelines.end(), created_pipelines.begin(), created_pipelines.end());
+
+        created_pipelines = create_pipelines(m_device->handle(), pipeline_layout.handle(), render_pass.handle(),
+            {size1, size2, size2, size1, size2, size2},
+            {capacity_vuid});
+        pipelines.insert(pipelines.end(), created_pipelines.begin(), created_pipelines.end());
+
+        // Test that multiple errors are generated when appropriate
+        created_pipelines = create_pipelines(m_device->handle(), pipeline_layout.handle(), render_pass.handle(),
+            {size1, size2, size1, size1, invalid_size1, size1, size1, size1},
+            {invalid_size_vuid, capacity_vuid});
+        pipelines.insert(pipelines.end(), created_pipelines.begin(), created_pipelines.end());
+
+        created_pipelines = create_pipelines(m_device->handle(), pipeline_layout.handle(), render_pass.handle(),
+            {size1, size2, invalid_size1, size2, size2, size2},
+            {invalid_size_vuid, capacity_vuid});
+        pipelines.insert(pipelines.end(), created_pipelines.begin(), created_pipelines.end());
+
+        // Test that filling up to the limits is still doable
+        created_pipelines = create_pipelines(m_device->handle(), pipeline_layout.handle(), render_pass.handle(),
+            {size1, size2, size1});
+        pipelines.insert(pipelines.end(), created_pipelines.begin(), created_pipelines.end());
+
+        // Then again, we should expect capacity issues with any attempt
+        created_pipelines = create_pipelines(m_device->handle(), pipeline_layout.handle(), render_pass.handle(),
+            {invalid_size1},
+            {invalid_size_vuid});
+        pipelines.insert(pipelines.end(), created_pipelines.begin(), created_pipelines.end());
+
+        created_pipelines = create_pipelines(m_device->handle(), pipeline_layout.handle(), render_pass.handle(),
+            {size1},
+            {capacity_vuid});
+        pipelines.insert(pipelines.end(), created_pipelines.begin(), created_pipelines.end());
+
+        created_pipelines = create_pipelines(m_device->handle(), pipeline_layout.handle(), render_pass.handle(),
+            {size2},
+            {capacity_vuid});
+        pipelines.insert(pipelines.end(), created_pipelines.begin(), created_pipelines.end());
+
+        // clang-format on
+
+        // Delete pipelines
+        for (auto pipeline : pipelines) {
+            vksc::DestroyPipeline(m_device->handle(), pipeline, nullptr);
+        }
+        pipelines.clear();
+    }
+}
+
+TEST_F(VkSCLayerTest, CreatePipelineCacheInvalidFlags) {
+    TEST_DESCRIPTION("vkCreatePipelineCache - missing required flags");
+    // NOTE: This test case implicitly tests the removed VUIDs: 00768, 007689
```

Comment:
is the first VUID (00768) missing a digit?

Reply:
Actually there's an excess "8" in the second VUID. Will fix it.